src/feedback/feedback.tpl

```diff
@@ -1,7 +1,9 @@
-<div id="{{id}}" class="feedback feedback-{{level}} {{#if popup}}popup{{/if}}">
+<div id="{{id}}" class="feedback feedback-{{level}} {{#if popup}}popup{{/if}}"
+     role="dialog" aria-describedby="{{dompurify msg}}" >
     <span class="icon-{{level}}"></span>
-    <div>
+    <div aria-live="{{dompurify msg}}">
         {{{dompurify msg}}}
     </div>
-    <span title="{{__ 'Close message'}}" class="icon-close" data-close=":parent .feedback"></span>
+    <button title="{{__ 'Close message'}}" class="icon-close" data-close=":parent .feedback"
```

Comment:
Probably we should keep `span` tag and add `tabindex="0"` and `role="button"`(?) attributes. This approach allow us to avoid possible css issues